extensions/vscode/src/views/logs.ts

```diff
@@ -362,8 +367,14 @@ export class LogsTreeLogItem extends TreeItem {
     msg: EventStreamMessage,
     state: TreeItemCollapsibleState = TreeItemCollapsibleState.None,
   ) {
+    if (msg.data.message) {
+      msg.data.message = msg.data.message.replaceAll("\n", " ");
```

Comment:
I still think we should be adjusting this in the backend, but totally fine leaving this in for now.

Reply:
I'm not sure. It really is a presentation layer decision.. for the CLI, a newline character is perfectly fine. I originally thought about showing it as an additional tree item in the view, but it was too complex with the way we are creating those. I'll add a triage item just so we can discuss and everyone can think about it... but I don't expect that we'll plan on having it resolved anytime soon.